si-id-epu50/README.md

```diff
@@ -0,0 +1,11 @@
+# SI-ID IOC conversion classes and scripts
+
+IOC classes and scripts for insertion devices conversions.
```

Comment:
# SI-ID EPU50 IOC classes and scripts

IOC classes and scripts for EPU50 insertion devices.

Reply:
deprecated